### PR TITLE
item type "DicomFragmentSequence" not generated in FellowOakDicom.Serialization.DicomXML class

### PR DESCRIPTION
**Pull request by ghost**
Fixes # .
XML Serialization issue when dataset element type is DicomFragmentSequence

#### Checklist
- [ ] The pull request branch is in sync with latest commit on the *fo-dicom/development* branch

#### Changes proposed in this pull request:
-Added new fix to handle the DicomFragmentSequence
-Quick new function GetBinaryBase64

### CODE REVIEW

**Comment by amoerie** [comment]
Hello @AshrafGanie and thanks for your first contribution!

I have the following remarks:

- While I applaud your enthusiasm, it is usually better to first create an issue so you can describe the problem, and we can have an interactive discussion about the fix. In this specific case though, I agree that the fix is rather straightforward and perhaps did not require a preceding discussion.
- You removed most of our pull request description template. Why? They exist for a reason, for example see my next point
- Could you please include an additional unit test that demonstrates your new code is working?

File: FO-DICOM.Core/Serialization/DicomXML.cs [hunk 1]
```diff
@@ -153,6 +171,12 @@ private static string GetBinaryBase64(DicomElement item)
             return Convert.ToBase64String(buffer.Data);
         }
 
+        private static string GetBinaryBase64(byte[] buffer)
```
Comment (amoerie) [inline FO-DICOM.Core/Serialization/DicomXML.cs]: If this signature was

`private static string GetBinaryBase64(IByteBuffer buffer)`

Then the method could be reused for regular DicomElement instances too.

Comment (MAshraf) [reply FO-DICOM.Core/Serialization/DicomXML.cs:174]: Hello @amoerie 
 Thanks for your response, let me create the unit test and will also check GetBinaryBase64(IByteBuffer buffer).

Comment (ghost) [reply FO-DICOM.Core/Serialization/DicomXML.cs]: Hello @amoerie 

Thanks for your response.  I will create required unit test.  Will also look into GetBinaryBase64(IByteBuffer buffer)

**Pull request closed without merging**